docs/admin/welcome/index.md

```diff
@@ -19,6 +19,15 @@ curl -L https://opencloud.eu/install | /bin/bash
 - Configuration – Customize and integrate with existing systems and storage
 - Maintenance – Learn how to update, backup and migrate
 
+## Documentation Versions
+
+Our documentation is versioned to ensure you always find the right information for your software version.
+
+- Rolling Version: Reflects the current development state. Changes are continuously added, but the documentation may be incomplete, as new features might not yet be documented. Ideal if you are using a pre-release or development build.
```

Comment:
could lead to confusion if rolling/prod refers only to the docs or the software. maybe just link here (if at all) https://docs.opencloud.eu/docs/admin/resources/lifecycle#advantage-of-the-lts-releases here it says doc on best effort - that should be enough CYA 😉